### PR TITLE
fix(demo): add warmup for wrong profiling result

### PR DESCRIPTION
**Pull request by tpoisonooo**
## Motivation

fix bad profiling result.

## Modification

add warmup

### CODE REVIEW

**Review by irexyc — APPROVED** [review]

File: demo/csrc/cpp/classifier.cxx [hunk 1]
```diff
@@ -26,11 +26,14 @@ int main(int argc, char* argv[]) {
   context.Add(mmdeploy::Device(FLAGS_device));
   context.Add(profiler);
 
+  // warmup
+  for (int i = 0; i < 20; ++i) {
+    classifier.Apply(img);
+  }
+
   // construct a classifier instance
```
Comment (irexyc) [inline demo/csrc/cpp/classifier.cxx:34]: should move these lines after line35

File: demo/csrc/cpp/rotated_detector.cxx [hunk 2]
```diff
@@ -27,6 +27,11 @@ int main(int argc, char* argv[]) {
   context.Add(mmdeploy::Device(FLAGS_device));
   context.Add(profiler);
 
+  // warmup
+  for (int i = 0; i < 20; ++i) {
+    detector.Apply(img);
+  }
+
```
Comment (irexyc) [inline demo/csrc/cpp/rotated_detector.cxx:34]: should move these lines after line36

File: demo/csrc/cpp/classifier.cxx (hunk 1 above)
Comment (tpoisonooo) [reply demo/csrc/cpp/classifier.cxx]: done.

File: demo/csrc/cpp/rotated_detector.cxx (hunk 2 above)
Comment (tpoisonooo) [reply demo/csrc/cpp/rotated_detector.cxx:37]: done

**Review by AllentDan — APPROVED** [review]

**Pull request merged**